CODE_OF_CONDUCT.md

```diff
@@ -58,11 +58,11 @@ Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported by contacting the project team at conduct@defold.se. All
 complaints will be reviewed and investigated and will result in a response that
 is deemed necessary and appropriate to the circumstances. The project team is
-obligated to maintain confidentiality with regard to the reporter of an incident.
+obligated to maintain confidentiality about the reporter of an incident.
```

Comment:
I'm not a native English speaker but I think _"maintain confidentiality about the reporter"_ sounds wrong and _"maintain confidentiality with regard to the reporter"_ sounds more correct.